tools/lambda-promtail/variables.tf

```diff
@@ -16,6 +16,12 @@ variable "bucket_names" {
   default     = []
 }
 
+variable "filter_suffix" {
+  type        = string
+  description = "Suffix for S3 bucket notification filter"
+  default     = ".log.gz"
```

Comment:
As an additional enhancement, perhaps this should become just `.gz` to capture both log types out of the box?

Reply:
does guard duty create two file types?